src/ApiService/ApiService/onefuzzlib/VmssOperations.cs

```diff
@@ -325,9 +352,9 @@ public class VmssOperations : IVmssOperations {
             //
             // https://docs.microsoft.com/en-us/azure/
             //   virtual-machine-scale-sets/use-spot#resource-manager-templates
-            vmssData.VirtualMachineProfile.EvictionPolicy = VirtualMachineEvictionPolicyTypes.Deallocate;
+            vmssData.VirtualMachineProfile.EvictionPolicy = VirtualMachineEvictionPolicyTypes.Delete;
             vmssData.VirtualMachineProfile.Priority = VirtualMachinePriorityTypes.Spot;
-            vmssData.VirtualMachineProfile.BillingMaxPrice = 1.0;
+            vmssData.VirtualMachineProfile.BillingMaxPrice = -1.0;
```

Comment:
-1.0 ?

Reply:
See comment above: "Setting max price to -1 means it won't be evicted because of price."